src/main/java/net/earthcomputer/clientcommands/features/VillagerCracker.java

```diff
@@ -74,12 +75,22 @@ public static void onSoundEventPlayed(ClientboundSoundPacket packet) {
         }
 
         switch (packet.getSound().value().getLocation().toString()) {
-            case "minecraft:entity.villager.ambient" -> ((IVillager) targetVillager).clientcommands_onAmbientSoundPlayed(packet.getPitch());
+            case "minecraft:entity.villager.ambient", "minecraft:entity.villager.trade" -> ((IVillager) targetVillager).clientcommands_onAmbientSoundPlayed(packet.getPitch());
             case "minecraft:entity.villager.no" -> ((IVillager) targetVillager).clientcommands_onNoSoundPlayed(packet.getPitch());
+            case "minecraft:entity.villager.yes" -> ((IVillager) targetVillager).clientcommands_onYesSoundPlayed(packet.getPitch());
             case "minecraft:entity.generic.splash" -> ((IVillager) targetVillager).clientcommands_onSplashSoundPlayed(packet.getPitch());
         }
     }
 
+    public static void onXpOrbSpawned(ClientboundAddExperienceOrbPacket packet) {
```

Comment:
this seems sketchy, can we add a distance from villager check?

Reply:
ok i can do it myself np

Reply:
Threre's already one in the same place as the sound distance check https://github.com/RealRTTV/clientcommands/pull/2/commits/47e53f20919b5800b4a3e5d870044a4324c3493d#diff-0189676b9ef31f3452bb9e11ebf3183c2b9069de3ca63fd8254cd872a909cd54R61